### ISSUE TITLE
Custom response instead of 'Unauthorised' response

### ISSUE THREAD

**Issue by robcalcroft** [jaredhanson/oauth2orize#140]
Would be great to be able to configure the response for when oauth2orize just sends `Unauthorized`, to be more like:
```json
{
    "status": "Unauthorized",
    "message": "Incorrect password"
}
```

**Comment by janjilek** [comment]
+1

**Comment by alessiodionisi** [comment]
+1

**Comment by akarshsatija** [comment]
+1

**Comment by josencv** [comment]
+1

**Comment by franck34** [comment]
+1

**Comment by aikssen** [comment]
+1

**Comment by rgrocha** [comment]
It's always a bad security practice to give tips why the user can't login.
It's up to you on the UI to show more human readable message, like "Bad username or password" or any other reason.

**Comment by m-harisqurashi** [comment]
+1

**Comment by robcalcroft** [comment]
@rgrocha I agree, wanted more for the response datatype to be customisable, ie JSON over HTML

**Comment by roshanr83** [comment]
Yeah I agree with @robcalcroft in my case I want to have same format of error response in my whole api so I want to customize that unauthorized message. Has anyone done it?
